docker/run_local_docker.sh

```diff
@@ -73,11 +73,19 @@ then
       rm -rf $PWD/docker-volume
     fi
 fi
+
+if [[ $VIRTUAL_ENV == "" ]]; 
+then 
+  echo "Please Use Virtual Environment and make sure to generate Pydantic Models"; 
+else
+  echo "Generating Pydantic Models"; 
+  make install_dev generate
```

Comment:
generating models via the script will reduce the number of failures on creation of ingestion image due to outdated models.